app/controllers/kaui/accounts_controller.rb

```diff
@@ -38,8 +38,18 @@ def pagination
     end
 
     formatter = lambda do |account|
+      account_close_link = ''
+      child_label = ''
+      if can? :create, Kaui::Account
```

Comment:
The permissions actually required are `entitlement:cancel`, `entitlement:pause_resume`, `tag:add` and `invoice:item_adjust`.

Maybe you can create some sort of helper to encapsulate this logic?